data_collection/gazette/spiders/ma/ma_duque_bacelar.py

```diff
@@ -1,13 +1,75 @@
-import datetime
+import urllib.parse
+from datetime import date
+from typing import List
 
-from gazette.spiders.base.siganet import BaseSiganetSpider
+import scrapy
 
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
 
-class MaDuqueBacelar(BaseSiganetSpider):
-    TERRITORY_ID = "2103901"
+
+class MaDuqueBacelar(BaseGazetteSpider):
     name = "ma_duque_bacelar"
-    start_date = datetime.date(2018, 6, 1)
-    allowed_domains = ["transparencia.duquebacelar.ma.gov.br"]
-    BASE_URL = (
-        "https://transparencia.duquebacelar.ma.gov.br/acessoInformacao/diario/diario"
-    )
+    TERRITORY_ID = "2103901"
+    allowed_domains = ["duquebacelar.ma.gov.br"]
+    start_urls = ["https://duquebacelar.ma.gov.br/transparencia/diario-oficial"]
+    start_date = date(2019, 8, 29)
+
+    def _extract_url(self, url_element: scrapy.Selector):
+        (raw_path,) = url_element.css("a")
+
```

Comment:
Se precisa só da primeira ocorrência, .get() é o suficiente

```suggestion
    def _extract_url(self, url_element):
        raw_path = url_element.css("a").get()

```

Reply:
no final das contas nem do .get() eu precisei porque constando apenas um elemento no retorno ele já não volta um array :+1: